less/home-page.less

```
@@ -53,9 +80,24 @@
     display: flex;
     justify-content: space-between;
 
+    @media(max-width: 500px) {
+      .column500px;
+      margin: 15px;
```

This is a no-no. You're setting margin to 15px on all sides. Hard coded pixels on the x-axis will make your website non-responsive. Would have been better to do something like:
```suggestion
      margin: 15px 1%;
```